core/php/Utilities/StringUtility.php

```diff
@@ -205,6 +205,35 @@ function az09($string, $preserve = "", $strToLower = TRUE) {
     $charGroup[] = array("2","²");
     $charGroup[] = array("3","³");
     $charGroup[] = array("and","&");
+    $charGroup[] = array("a","А","а");
+    $charGroup[] = array("b","Б","б");
+    $charGroup[] = array("v","В","в");
+    $charGroup[] = array("g","Г","г");
+    $charGroup[] = array("d","Д","д");
+    $charGroup[] = array("e","Е","е","Ё","ё");
+    $charGroup[] = array("zh","Ж","ж");
+    $charGroup[] = array("z","З","з");
+    $charGroup[] = array("i","И","и","Ы","ы");
+    $charGroup[] = array("y","Й","й","Ъ","ъ");
+    $charGroup[] = array("k","К","к");
+    $charGroup[] = array("l","Л","л");
+    $charGroup[] = array("m","М","м");
+    $charGroup[] = array("n","Н","н");
+    $charGroup[] = array("o","О","о");
+    $charGroup[] = array("p","П","п");
+    $charGroup[] = array("r","Р","р");
+    $charGroup[] = array("s","С","с");
+    $charGroup[] = array("t","Т","т");
+    $charGroup[] = array("u","У","у");
+    $charGroup[] = array("f","Ф","ф");
+    $charGroup[] = array("h","Х","х");
+    $charGroup[] = array("ts","Ц","ц");
+    $charGroup[] = array("ch","Ч","ч");
+    $charGroup[] = array("sh","Ш","ш","Щ","щ");
+    $charGroup[] = array("`","Ь","ь");
```

Comment:
the purpose of az09 is to have some fuzzyness/tolerance within different spelling of artists/genres/labels.
So in my opinion the backtick character should be removed

**for example**
```
Stereo Mc's
Stereo Mc´s
```
should be unified for internal handling with `stereomcs`

please remove this line.